flights/berks_ceh_incomes_flight.yaml

```
@@ -44,15 +44,15 @@ entityDefinitions:
       ol.value: 
         type: "ol.value"
         column: "TotalMonthlyIncome"
+        transforms:
+        - !<transforms.PrefixTransform>
+          prefix: "$"
```

can you take out all of the $ prefixes? It won't work because this data type is a double. It has to be a number only. It will be implied that it is money, or if you want you can write that into the dataset description.
```suggestion
          prefix: "$"
```